Jenkinsfile.integration

```diff
@@ -244,6 +245,14 @@ pipeline {
                         reportName: 'ARA Report'
             ]
             logParser projectRulePath: 'log-parser-plugin-rules.txt', showGraphs: true, useProjectRule: true, parsingRulesPath: null
+            script {
+                try {
+                    sh "./run.sh parse_tempest_ci"
```

Comment:
What's the behavior when tempest pod is not deployed (because of some site update error before this pod is deployed and tempest executed) as its 'always' executed.

Reply:
I changed this to only trigger on tempest step success :+1: